niworkflows/interfaces/registration.py

```diff
@@ -24,24 +46,26 @@ class FLIRTRPT(nrc.RegistrationRC, fsl.FLIRT):
     def _post_run_hook(self, runtime):
         self._fixed_image = self.inputs.reference
         self._moving_image = self.aggregate_outputs().out_file
-
+        NIWORKFLOWS_LOG.info('Report - setting fixed (%s) and moving (%s) images',
+                             self._fixed_image, self._moving_image)
 
 class ApplyXFMInputSpecRPT(nrc.RegistrationRCInputSpec,
                            fsl.preprocess.ApplyXFMInputSpec):
     pass
 
-class ApplyXFMRPT(FLIRTRPT):
-    ''' ApplyXFM is a wrapper around FLIRT. ApplyXFMRPT is a wrapper around FLIRTRPT.'''
-    input_spec = ApplyXFMInputSpecRPT
 
-class ANTSRegistrationInputSpecRPT(nrc.RegistrationRCInputSpec,
-                                   ants.registration.RegistrationInputSpec):
-    pass
+# class ApplyXFMRPT(FLIRTRPT):
+#     ''' ApplyXFM is a wrapper around FLIRT. ApplyXFMRPT is a wrapper around FLIRTRPT.'''
+#     input_spec = ApplyXFMInputSpecRPT
 
-class ANTSRegistrationOutputSpecRPT(nrc.ReportCapableOutputSpec,
-                                    ants.registration.RegistrationOutputSpec):
-    pass
+# class ANTSRegistrationInputSpecRPT(nrc.RegistrationRCInputSpec,
+#                                    ants.registration.RegistrationInputSpec):
+#     pass
+
+# class ANTSRegistrationOutputSpecRPT(nrc.ReportCapableOutputSpec,
+#                                     ants.registration.RegistrationOutputSpec):
+#     pass
 
-class ANTSRegistrationRPT(nrc.RegistrationRC, ants.Registration):
-    input_spec = ANTSRegistrationInputSpecRPT
-    output_spec = ANTSRegistrationOutputSpecRPT
+# class ANTSRegistrationRPT(nrc.RegistrationRC, ants.Registration):
+#     input_spec = ANTSRegistrationInputSpecRPT
+#     output_spec = ANTSRegistrationOutputSpecRPT
```

Comment:
That was there before but the actual ANTS interface is not yet implemented. I will address this in a PR for #54 .